templates/typo3/files/README.md

```diff
@@ -22,6 +22,8 @@ TYPO3 is a Content Management System (CMS) built in PHP.
 
 2. The installer will create an administrator account with username/password `admin`/`password`.  **You need to change this password immediately. Not doing so is a security risk**.
 
+3. Enable the `pixelant/pxa-lpeh` plugin.  It is already installed by default but must be enabled manually.  This plugin changes TYPO3's 403/404 page handling to avoid a self-request HTTP request that can cause a race condition and deadlocks in some situations.
```

Comment:
```suggestion
3. Enable the `pixelant/pxa-lpeh` plugin.  It is already installed by default but must be enabled manually.  This plugin changes TYPO3's 403/404 page handling to avoid a self-request HTTP request that can cause a race condition and deadlocks in some situations. See the [TYPO3 documentation](https://docs.platform.sh/frameworks/typo3.html#avoiding-deadlock) for more information.
```